include/Reader/readerir.h

```diff
@@ -1139,6 +1143,18 @@ class GenIR : public ReaderBase {
     return makeLoad(Address, IsVolatile, false);
   }
 
+  /// \brief Create a call or invoke instruction
+  ///
+  /// The call is inserted at the LLVMBuilder's current insertion point.
+  ///
+  /// \param Callee    Target of the call
+  /// \param Args      Arguments to pass to the callee
+  /// \param MayThrow  True if the callee may raise an exception
+  /// \returns         A \p CallSite wrapping the CallInst or InvokeInst
+  llvm::CallSite makeCall(llvm::Value *Callee,
+                          llvm::ArrayRef<llvm::Value *> Args,
+                          bool MayThrow = true);
```

Comment:
Having a default value for `MayThrow` seems like the wrong thing to me. I'd much rather require that MayThrow be explicitly specified at all callsites for the sake of clarity (and to encourage some thought about whether or not the target might actually throw).